### ISSUE TITLE
WCAG 4.1.2: Ensures buttons have discernible text (#user_menu)

### ISSUE THREAD

**Issue by maxvisser** [teamforus/forus#2306]
#### Issue
`Ensures buttons have discernible text` ([button-name](https://accessibilityinsights.io/info-examples/web/button-name))
#### Target application
[Uw tegoed](https://berkelland.staging.forus.io/vouchers)
#### Element path
#user_menu
#### Snippet
`<button class="auth-user-caret" id="user_menu" aria-expanded="false"><em class="mdi mdi-chevron-down" ng-class="$ctrl.userMenuOpened ? 'mdi-chevron-up' : 'mdi-chevron-down'"></em></button>`
#### How to fix
    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"
#### Environment
Microsoft Edge version 90.0.818.66
This accessibility issue was found using Accessibility Insights for Web 2.26.1 (axe-core 4.2.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Comment by maxvisser** [comment]
VOUCHERS NIET BESCHIKBAAR VIA TAB
KAN NIET UITLOGGEN

skiplink: https://www.roxit.nl/Themas/Commonground
